elastic/integrations
# Add dimension fields to support TSDB enablement

- Enhancement

## What does this PR do?

- TSDB Enablement for Cockroachdb.

## Checklist


## How to test this PR locally

Download, Install, configure cockroachdb
Configure the cockroachdb integration
Verify the dashboard is loaded correctly without missing data except panels reporting errors.

## Related issues

- Relates #5410
